NEWS.md

```diff
@@ -1,5 +1,35 @@
 MXNet Change Log
 ================
+## 0.11.0
```

Comment:
should we include the Keras and CoreML section as well from https://cwiki.apache.org/confluence/display/MXNET/v0.11.0+Release+Notes ?